### PR TITLE
COP-11820 Added a withdrawn tab to targets list

### PR DESCRIPTION
**Pull request by FrancisBry**
Added a withdrawn tab to the RoRo v2 and airpax targets list

image

### CODE REVIEW

File: client/src/pages/targets/components/TargetListItemOptions.jsx [hunk 1]
```diff
@@ -169,8 +169,8 @@ export const OutcomeButton = ({ selectedTab, targetStatus, onClick, permission }
   return null;
 };
 
-export const DetailsButton = ({ onClick, permission }) => {
-  if (!permission) {
+export const DetailsButton = ({ selectedTab, onClick, permission }) => {
```
Comment (charles474) [inline client/src/pages/targets/components/TargetListItemOptions.jsx:172]: Missing test which checks the selected tab.

Comment (FrancisBry) [reply client/src/pages/targets/components/TargetListItemOptions.jsx:172]: Added it

File: client/src/pages/targets/adapters/row/targetRow.js [hunk 2]
```diff
@@ -17,8 +17,31 @@ const isChecked = (checkedRadioBtnName, radioBtnName) => {
   return checkedRadioBtnName === radioBtnName;
 };
 
+const toRadioColumn = (status, targetId, name, onChange, checkedRadioBtnName) => {
+  const columnClasses =
```
[the remark this answers is not in the document]
Comment (FrancisBry) [reply client/src/pages/targets/adapters/row/targetRow.js]: Changed it to be like this

**Review by charles474 — APPROVED** [review]
LGTM

**Pull request merged**